package.json

```diff
@@ -15,7 +15,7 @@
     "url": "https://github.com/readmeio/metrics-sdks.git"
   },
   "engines": {
-    "node": ">=10"
+    "node": "^12 || ^14 || ^16"
```

Comment:
Sorry can you update this in `packages/node/package.json` too?

Reply:
ah good point

Reply:
done